### DISCUSSION TITLE
RSS-GPT 使用指南 - 1nan

### DISCUSSION THREAD

**Discussion by giscus** [yinan-c/pages-comment#8] [Announcements]
# RSS-GPT 使用指南 - 1nan

https://yinan.me/rss-gpt-manual-zh.html

**Comment by socekin** [comment]
给项目点个赞，想了解一下过个RSS源去除重复内容的规则是什么？这部分会消耗OpenAI的Token吗？

**Reply by yinan-c** [reply]
Hi，目前过滤功能的实现，就是先抓取新文章的标题，url 和描述，然后用一个正则表达式来过滤掉符合条件的文章。不会使用 OpenAI API，只有同时设定了 OpenAI API 和总结条目数才会调用总结功能消费  API token，所以你也可以独立使用过滤功能。

**Comment by lmm214** [comment]
我看代码里有 `OPENAI_PROXY` 这个参数，加了后好像没启用

**Comment by YAHKLI** [comment]
在哪里查看新生成rss源呢？
